draft-cenzano-moq-media-interop.md

```diff
@@ -213,45 +186,47 @@ Seq Id
 
 Monotonically increasing counter for this media track
 
-Presence Flags
-
-Indicates which of the following fields will be present
-
 PTS Timestamp
 
-Present only if `PresenceFlags & 0x1 > 0`, if not present copy from latest
-received.
 Indicates PTS in timebase
 TODO: Varint does NOT accept easily negative, so it could be challenging to
 encode at start (priming)
 
+Presence Flags
+
+Indicates which of the following fields will be present
+
 Duration
 
-Present only if `PresenceFlags & 0x10 > 0`, if not present copy from latest
-received.
+Present only if `PresenceFlags & 0x1 > 0`, if not present copy from latest
+received. It is recommeneded to send this info periodically
```

Comment:
"copy from latest" is weird because everything is its own stream, and things can be reordered.  It would be safer to say "copy from the largest object received with a `Seq ID` less than the `Seq ID` of this object.

However, that still leaves the possibility that something is wrong eg:

```
seq=1: settings1
seq=2: no-settings
seq=3: settings2
seq=4: no-settings
```
What if object 4 arrives before object 3?  Does it get decoded with the wrong settings?

 Maybe this is a good place to use groups -- eg encoder cuts a new group when settings change, and we say "copy from any object earlier in the group".
 
 So it would be:
 
```
group=0, obj=0, seq=1: settings1
group=0, obj=1, seq=2: no-settings
group=1, obj=0, seq=3: settings2
group=1, obj=1, seq=4: no-settings
```

Reply:
This may also mean we can omit all the flags on objects > 0.

Reply:
I think my next change covers all of tha ^^